susemanager-utils/susemanager-sls/src/tests/test_grains_cpuinfo.py

```diff
@@ -82,3 +84,22 @@ def test_cpusockets_lscpu():
                 assert 'cpusockets' in out
                 assert out['cpusockets'] == 1
 
+
+@pytest.mark.parametrize("arch", ["x86_64", "aarch64", "s390", "ppc64"])
+def test_cpusockets_cpu_data(arch):
+    '''
+    Test lscpu -J data extraction function.
+
+    :return:
+    '''
+    cpuinfo.log = MagicMock()
+    with patch('src.modules.udevdb._which_bin', MagicMock(return_value="/bogus/path")):
```

Comment:
I think this is the wrong path that is getting patched, `grains.cpuinfo` imports `which_bin` from either `salt.utils.path` or `salt.utils`.

Reply:
well... that one is a copy/paste from the other test functions that where present in the same file. Interestingly enough whatever the path is doesn't change the test result since the salt imports are mocked. `_which_bin` will be an uninitialized `MagicMock` which is not `None`.